source/composeK.js

```diff
@@ -39,5 +38,8 @@ export default function composeK() {
   }
   var init = Array.prototype.slice.call(arguments);
   var last = init.pop();
+  if (init.length === 0) {
+    return last;
+  }
```

Comment:
Even nicer, I think:

```javascript
return init.length === 0 ?
  last :
  compose(compose.apply(this, map(chain, init)), last);
```

Expressions are just about always preferable to statements. :)

Reply:
I was totally going to do that;) then I saw the if statement up top and resorted to conform to my imaginary standard that Ramda team might have had...